include/powsybl/iidm/converter/Parameter.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_CONVERTER_PARAMETER_HPP
+#define POWSYBL_IIDM_CONVERTER_PARAMETER_HPP
+
+#include <initializer_list>
+#include <set>
+#include <string>
+#include <vector>
+
+#include <boost/any.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace converter {
+
+class Parameter {
+public:
+    enum class ParameterType : char {
+        BOOLEAN,
+        STRING,
+        STRING_LIST
+    };
+
+public:
+    Parameter(std::string&& name, const ParameterType& type, std::string&& description, std::string&& defaultValue);
+
+    Parameter(std::string&& name, const ParameterType& type, std::string&& description, bool defaultValue);
+
+    Parameter(std::string&& name, const ParameterType& type, std::string&& description, std::set<std::string>&& defaultValue);
+
+    ~Parameter() = default;
+
+    Parameter(const Parameter&) = default;
+
+    Parameter(Parameter&&) noexcept = default;
+
+    Parameter& operator=(const Parameter&) = default;
+
+    Parameter& operator=(Parameter&&) = delete;
```

Comment:
Why do you delete this one? Maybe it's acceptable to declare it as default

Reply:
You suggested it here https://github.com/powsybl/powsybl-iidm4cpp/pull/106#discussion_r430050939.
I applied the modification, but forgot to ask you why you wanted to delete it => reverted